apps/omg_watcher_rpc/lib/web/controllers/transaction.ex

```diff
@@ -39,7 +39,7 @@ defmodule OMG.WatcherRPC.Web.Controller.Transaction do
   Retrieves a list of transactions
   """
   def get_transactions(conn, params) do
-    with {:ok, constraints} <- Validator.Constraints.parse(params) do
+    with {:ok, constraints} <- Validator.TransactionConstraints.parse(params) do
       InfoApiTransaction.get_transactions(constraints)
       |> api_response(conn, :transactions)
```

Comment:
### 
```suggestion
      api_response(InfoApiTransaction.get_transactions(constraints), conn, :transactions)
```

Reply:
I changed to 2 pipes instead. More readable.